turbo/app/snapshots_cmd.go

```diff
@@ -949,6 +950,14 @@ func doMeta(cliCtx *cli.Context) error {
 		}
 
 		log.Info("meta", "distances(*100K)", fmt.Sprintf("%v", distances))
+	} else if strings.HasSuffix(fname, ".kvi") || strings.HasSuffix(fname, ".idx") || strings.HasSuffix(fname, ".efi") || strings.HasSuffix(fname, ".vi") {
```

Comment:
.kvi is not being used right now. Or is this set of changes going to replace .kvei/.bt for .kvi?

Reply:
I will make `.kvi` default for `commitment.kv` in https://github.com/erigontech/erigon/pull/13276